Packages/Components/Sources/AssetListView.swift

```diff
@@ -82,7 +82,7 @@ public struct AssetListItemView: View {
     public var body: some View {
         HStack {
             imageView
-            HStack(alignment: .center) {
+            HStack {
```

Comment:
make sure this does not break other places

Reply:
Sure. By default HStack initialise with alignment: center.  HStack(alignment: .center) - not necessary to write